src/qt/forms/importkeysdialog.ui

```diff
@@ -0,0 +1,226 @@
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QTabWidget" name="tabWidget">
+     <property name="currentIndex">
+      <number>0</number>
+     </property>
+     <widget class="QWidget" name="tabImportPrivateKey">
+      <attribute name="title">
+       <string>&amp;Import Private Key</string>
+      </attribute>
+      <layout class="QVBoxLayout" name="verticalLayout_Display">
+       <item>
+        <layout class="QHBoxLayout" name="horizontalLayout_1_ImportPrivateKey">
+         <item>
+          <widget class="QLabel" name="privateKeyWidgetLabel">
+           <property name="text">
+            <string>Private Key:</string>
+           </property>
+           <property name="textFormat">
+            <enum>Qt::PlainText</enum>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QLineEdit" name="privateKey">
+           <property name="toolTip">
+            <string>Private key to import into your wallet</string>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </item>
+       <item>
+        <layout class="QHBoxLayout" name="horizontalLayout_2_ImportPrivateKey">
+         <item>
+          <widget class="QLabel" name="privateKeyLabelWidgetLabel">
+           <property name="toolTip">
+            <string>Label for this private key in your wallet</string>
+           </property>
+           <property name="text">
+            <string>Label (optional):</string>
+           </property>
+           <property name="buddy">
+            <cstring>privateKeyLabel</cstring>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QLineEdit" name="privateKeyLabel">
+           <property name="toolTip">
+            <string>Label for this private key in your wallet</string>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </item>
+       <item>
+        <widget class="QCheckBox" name="rescanCheckBox">
+         <property name="toolTip">
+          <string extracomment="Rescan the blockchain on import (disabled when running in pruned mode)."/>
+         </property>
+         <property name="text">
+          <string>Rescan</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QLabel" name="privateKeyImportTextMessage">
+         <property name="text">
+          <string></string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <spacer name="verticalSpacer_ImportPrivateKey">
+         <property name="orientation">
+          <enum>Qt::Vertical</enum>
+         </property>
+         <property name="sizeHint" stdset="0">
+          <size>
+           <width>20</width>
+           <height>40</height>
+          </size>
+         </property>
+        </spacer>
+       </item>
+      </layout>
+     </widget>
+    </widget>
```

Comment:
matching widget closing tag for `QTabWidget`

Reply:
Done, thanks!